redbot/cogs/trivia/data/lists/worldcup.yaml

```
@@ -432,3 +456,249 @@ In 2018, Italy failed to qualify after losing to *who* 1-0?:
 - Sweden
 In 2002, Netherlands failed to qualify after losing to Republic of Ireland. What was the goal difference in this qualification decider?:
```

```suggestion
In 2002, Netherlands failed to qualify after losing to the Republic of Ireland. What was the goal difference in this qualification decider?:
```
While you're touching this